frontend/app/src/view/ChatPanel.tsx

```diff
@@ -17,6 +19,33 @@ const LeaveButton = ({ onClick }: { onClick: () => void }) => {
   )
 }
 
+interface ExtraOptionProps {
+  socket: ChatSocket
+  roomInfo: { bool: boolean; roomId: number; roomType: string }
+}
+
+const ExtraOptionPerRoom = ({ socket, roomInfo }: ExtraOptionProps) => {
+  const [isOwner, setIsOwner] = useState(false)
+  const { data: me, isSuccess: meOk } = useApiQuery<User>(['user', 'me'])
+  const { data: users, isSuccess: usersOk } = useApiQuery<ChatUser[]>(
+    ['chat', roomInfo.roomId, 'list'],
+    { enabled: meOk },
```

Comment:
```suggestion
    { enabled: meOk },
```
생각해보니까 이부분이 `roomId`를 `me.uid`로 착각해서 조건으로 넣었던 거를 까먹고 안 지우고 있었네요...

Reply:
아 어쩐지 ㅋㅋ 무슨 관련이 있나 했네요